### DISCUSSION TITLE
Achievement / Badges, just why??

### DISCUSSION THREAD

**Discussion by Rellotscrewdriver** [community/community#18587] [Other Feature Feedback, Questions, & Ideas] [answered]
**NOTE:** I'll be mostly referring badges/achievements as achievements in this post

who thought it was a good idea to add achievements?
as this will become a VERY good way for people to spam it to gain achievements. and I suspect these achievements will feel cluttered on the left side of the profile page once someone achieved a lot of badges.

if the majority of people like this feature, I hope you add an option to disable showing achievements to anyone(yourself and other people) on the profile page 

and to whoever proposed this: github is NOT a video game! I'm fine having achievements in games but not in platforms where people download open-source software or/and contribute to it! not every software needs achievements fool

**Comment by AlexWinder** [comment, marked as answer]
Achievements have been on profiles for some time now, only recently have they been extended. That said you can disable them from being displayed on your profile at https://github.com/settings/profile

Scroll to the bottom and you can disable them.

![image](https://user-images.githubusercontent.com/24867605/173617431-40be10a8-1ced-4237-a3e8-acce4b064cff.png)

**Reply by rattrayalex** [reply]
Thank you for sharing this – note that you have to click "Update preferences" to apply the change. I filed an issue about this here: https://github.com/github-community/community/discussions/18671

**Reply by AlexWinder** [reply]
I'm not so sure that I would call that an issue, this behaviour is no different to lots of websites where you need to confirm your action before it's made. I think the issue is more of people complaining about the names/styling of the achievements rather than the struggle to hide them.

**Reply by rattrayalex** [reply]
It isn't a _bug_, but I do think it's a problem. I'll add more detail in #18671

**Reply by stevenxxiu** [reply]
This doesn't solve the issue. Need an option to disable everyone's achievements from being shown.

**Comment by stevenxxiu** [comment]
Here's a *uBlock Origin* filter to block this junk:

    github.com##.d-block:has(a[href*="?tab=achievements"])
    github.com##.d-md-block:has(a[href*="?tab=achievements"])

Congratulations to whichever *Microsoft* manager came up with this idea. Like the idea of training *Copilot* on open source code, it's just gonna slowly move everyone away from *GitHub*.

**Comment by abenhamdine** [comment]
> and to whoever proposed this: github is NOT a video game!

Can't agree more. Too much gamification kill gamification 😑
